src/components/client-registration.tsx

```diff
@@ -106,7 +106,7 @@ const  ClientRegistration: React.FC<{config: AuthConfig, updateRegistrationSucce
             "appName": appName,
             "callbackUrl": callbackUrl ? callbackUrl + ', https://tokens.ala.org.au': 'https://tokens.ala.org.au',
             "resourceOwner": resourceOwner,
-            "scopes": scopes ? scopes: 'openid email profile roles',
+            "scopes": scopes ? scopes: 'openid email profile ala/roles',
```

Comment:
This change needs to be done once the cognito is released? Until that token generation happens with CAS.